src/cloudant/account.py

```diff
@@ -322,43 +329,30 @@ def __setitem__(self, key, value, remote=False):
             value.create()
         super(CouchDB, self).__setitem__(key, value)
 
-
 class Cloudant(CouchDB):
     """
-    _Cloudant_
-
-    Object that encapsulates a cloudant account,
-    handling top level user API calls, database
-    creation, token generation et al.
+    Encapsulates a Cloudant client, handling top level user API calls,
+    database creation, token generation et al.
```

Comment:
I changed this to be a more generic description to encompass functionality related to session management and database management.  That way we don't need to continue to add to the description as the library matures.  Done in b19046696b22d6b3fb89e04d7d4b7a86ca7d9c41.